client/src/mock/issuersData.js

```diff
@@ -0,0 +1,277 @@
+export const issuersData = [
```

Comment:
It would be really good to make a script of this. 

Let's consider adding a script to populate the db with this data. We'll have to decide on a data model or consider what is already existing. 

Thanks for this.